package.xml

```diff
@@ -13,8 +13,8 @@
   <build_depend>git</build_depend>
   <build_depend>doxygen</build_depend>
   <depend>eigen</depend>
-  <!-- There is no key for simde yet - deactivating for now. -->
-  <!-- <depend>simde</depend> -->
+  <depend condition="$ROS_DISTRO == humble">simde</depend>
+  <depend condition="$ROS_DISTRO == rolling">simde</depend>
```

Comment:
Thank you for the updates. Is it possible to use `depend="$ROS_DISTRO =='humble' || $ROS_DISTRO == 'rolling'`? And to leave a comment saying simde is not available for Noetic/20.04 or only available for 22.04 and following?

Reply:
> Thank you for the updates. Is it possible to use depend="$ROS_DISTRO =='humble' || $ROS_DISTRO == 'rolling'?
Yes, but considering upcoming ros2 release, is it better to use `condition="$ROS_DISTRO != noetic or $ROS_DISTRO != foxy"`? Do you have any thoughts?

> And to leave a comment saying simde is not available for Noetic/20.04 or only available for 22.04 and following?
Sure!

Reply:
I am not sure if someone may not use this on other (even EOL) distros - so explicitly enabling where available might be the better choice? I am open for either though

Reply:
Thank you for your opinion.  I choose `condition="$ROS_DISTRO == humble or $ROS_DISTRO == rolling"`.

Reply:
@wxmerkt Please check [the change](https://github.com/Simple-Robotics/proxsuite/pull/178/commits/ce123e14a1d2eac2dbaf2476df3cfee031a9f657).

Reply:
Perfect, thank you!